Add documentation for `gp timeout show` command (#2292)
gitpod-io/website: gitpod/docs/command-line-interface.md

The default timeout, and the ability to extend a workspace timeout depends on your [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams).

### show

> Shows the current workspace's timeout. Depending on your [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams) this may be extendable using `gp timeout extend`.

Reviewer: **Suggestion:** Swap "this" for "workspace timeout". 

**Suggestion:** Flip the wording for subject->caveat, rather than caveat->subject.

```suggestion
Shows the current workspace's timeout. The workspace timeout may be extended using `gp timeout extend` if available on the users [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams).
```

> Shows the current workspace's timeout. The workspace timeout may be extended using `gp timeout extend` if available on the user's [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams).

Reviewer: **thought:** should we add here a note / example of the output format?

Author: I'd say no, for consistency. The other commands in this doc don't show example output.

Reviewer: Thought was really about making a note about the `m` suffix formatting, but it's a minor thing.

Reviewer: Should be intuitive enough, I hope!
